app/src/main/java/com/tohami/newsapi/adapter/NewsAdapter.java

```diff
@@ -0,0 +1,104 @@
+package com.tohami.newsapi.adapter;
```

Comment:
Delete this package, any adapter should be included in its component's package

Reply:
done